### DISCUSSION TITLE
Focus on making Dialer and SMS apps work extremely well cross-profile

### DISCUSSION THREAD

**Discussion by seankhl** [FossifyOrg/General-Discussion#36] [Ideas]
One pain of using profiles is that the Dialer and SMS apps behave suboptimally, especially with regard to notifications, when used between profiles. For example, the current Dialer app sometimes sends a notification in the main profile when in another profile when receiving a call, even though calling can be initiated from the current profile too. And incoming SMS messages will always trigger a notification in the main profile, even though (sometimes after waiting a few seconds) the SMS can be viewed in the current profile too.

Very few apps prioritize correct functionality here, and it would be fantastic for these two already excellent apps to support this; pretty much anyone who uses profiles would benefit greatly from it.

**Comment by naveensingh** [comment]
@seankhl 
Issue templates are ready, we are still setting up labels and such but it would be best if you create separate issues in the Dialer and SMS Messenger repo with specific steps to reproduce your mentioned issues.

I usually don't complain about things but SMT had this problem. Unclear/unstructured issues without enough information meant the developers had to spend double or triple the time it would actually take to reproduce an issue. Sometimes, some important issues took hours (if not days) before they could be reproduced because the reporters weren't available anymore.

**Reply by seankhl** [reply]
I will post the issues with full information this weekend! Thanks for being on top of this!
